stdlib/REPL/test/docview.jl

```diff
@@ -5,9 +5,9 @@ import REPL
 @test startswith(let buf = IOBuffer()
         Core.eval(Main, REPL.helpmode(buf, "α"))
         String(take!(buf))
-    end, "\"α\" can be typed by \\alpha<tab>\n")
+    end, "\"α\" can be typed by \:alpha<tab>\n")
 
 @test startswith(let buf = IOBuffer()
         Core.eval(Main, REPL.helpmode(buf, "🐨"))
         String(take!(buf))
-    end, "\"🐨\" can be typed by \\coala<tab>\n")
+    end, "\"🐨\" can be typed by \:koala<tab>\n")
```

Comment:
You still need a double backslash, to [escape the backslash](https://xkcd.com/1638/).  And there needs to be a colon at the end `koala` as well as the beginning.

That is, it should be `\\:koala:`